classes/village/VillageManager.php

```diff
@@ -2255,10 +2585,10 @@ private static function setNewRelations(System $system, int $initiator_village_i
                     $relation_name = "Neutral";
                     break;
                 case VillageRelation::RELATION_ALLIANCE:
-                    $relation_name = VillageManager::VILLAGE_NAMES[$initiator_village_id] . "-" . VillageManager::VILLAGE_NAMES[$recipient_village_id] . " Alliance";
+                    $relation_name = VillageManager::VILLAGE_NAMES[$initiator_village_id] . " - " . VillageManager::VILLAGE_NAMES[$recipient_village_id] . " Alliance";
                     break;
                 case VillageRelation::RELATION_WAR:
-                    $relation_name = VillageManager::VILLAGE_NAMES[$initiator_village_id] . "-" . VillageManager::VILLAGE_NAMES[$recipient_village_id] . " War";
+                    $relation_name = VillageManager::VILLAGE_NAMES[$initiator_village_id] . " - " . VillageManager::VILLAGE_NAMES[$recipient_village_id] . " War";
                     break;
                 default:
                     break;
```

Comment:
At 2594 lines I think it's time to start thinking about if we can split this into two classes